### DISCUSSION TITLE
HTML Scale Labels

### DISCUSSION THREAD

**Discussion by Gfeigel** [fbdesignpro/sweetviz#81] [General]
Please see attachment. The labels are cut off. My screen resolution is 1920x1080. Is there something I need to change?

[Sweetviz Labels.pdf](https://github.com/fbdesignpro/sweetviz/files/6065883/Sweetviz.Labels.pdf)

**Comment by fbdesignpro** [comment]
Hi @Gfeigel, thank you for the report, I will follow-up in the issue you raised.

**Comment by fbdesignpro** [comment]
For anyone following this thread, this was caused by matplotlib not following the configuration commands sweetviz was using. The root cause was not found yet, suspecting some sort of environment/install configuration overriding sweetviz (?).

**Reply by panstav1** [reply]
Apparently, for anyone interested in this as I was, the problem was introduced indeed from Matplotlib, more specifically, on the box inches. The working Matplotlib version that worked in my case was 3.4.3. It seems sth introduced at 3.5.0 sets other box_inches and padding is done afterwards in this library.

My current setup was MLFlow Runs in a project so as to log artifact HTML report.
